src/nvim/CMakeLists.txt

```
@@ -53,6 +53,8 @@ set(UNICODE_DIR ${PROJECT_SOURCE_DIR}/unicode)
 set(GENERATED_UNICODE_TABLES ${GENERATED_DIR}/unicode_tables.generated.h)
 set(VIM_MODULE_FILE ${GENERATED_DIR}/lua/vim_module.generated.h)
 set(VIM_MODULE_SOURCE ${PROJECT_SOURCE_DIR}/src/nvim/lua/vim.lua)
+set(SHARED_MODULE_SOURCE ${PROJECT_SOURCE_DIR}/runtime/lua/vim/shared.lua)
+set(INSPECT_MODULE_SOURCE ${PROJECT_SOURCE_DIR}/runtime/lua/vim/inspect.lua)
```

let's prefix these with `LUA_`. (up to you if you want to also rename `VIM_MODULE_*` or not...)

```suggestion
set(LUA_SHARED_MODULE ${PROJECT_SOURCE_DIR}/runtime/lua/vim/shared.lua)
set(LUA_INSPECT_MODULE ${PROJECT_SOURCE_DIR}/runtime/lua/vim/inspect.lua)
```

---

Personally I'm wondering why we can't just inline those into the area they're being used since they are only used once. But I agree about consistent naming otherwise

---

Oh wait they are used more than once. retracted.